spec/acceptance/server/grant_spec.rb

```diff
@@ -0,0 +1,190 @@
+require 'spec_helper_acceptance'
+
+describe 'postgresql::server::grant:', :unless => UNSUPPORTED_PLATFORMS.include?(fact('osfamily')) do
+
+  db = 'grant_priv_test'
+  owner = 'psql_grant_priv_owner'
+  user = 'psql_grant_priv_tester'
+  password = 'psql_grant_role_pw'
+
+  pp_setup = <<-EOS.unindent
```

Comment:
Variables should not be declared outside of tests. If it is needed by multiple tests then a `let()` should be used. Yes, our code does the same thing for `db`/`owner`/`user`/`password` and it's wrong and needs fixing :)